wordpress-mobile/gutenberg-mobile
# Experiment: Move unselected block accessibility handling to block-holder

Alternative to https://github.com/wordpress-mobile/gutenberg-mobile/pull/936
Gutenberg PR https://github.com/WordPress/gutenberg/pull/15225

> This PR moves accessibility handling of unselected blocks to the BlockHolder

It updates e2e tests as well which were broken with this updates (we rely on the accessibility label to find blocks).

### Testing Instructions

- Try navigating blocks on Android with Talkback on
- Try navigating blocks on iOS with VoiceOver on

For instance:

- An heading block should have all the following information spoken:
"Heading Block. Row X. <content>. Button"

- A paragraph block should have all the following information spoken:
"Paragraph Block. Row X. <content>. Button"

- An empty image block should have all the following information spoken:
"Image Block. Row X. Empty. Button"

- A non-empty image block with a caption should have all the following information spoken:
"Image Block. Row X. <content of the caption>. Button"
